src/Pdk/Product/Repository/WcPdkProductRepository.php

```diff
@@ -112,7 +117,13 @@ private function getWcProduct($identifier): WC_Product
             $product = $identifier;
         } else {
             $product = $this->retrieve('wc_product' . $identifier, function () use ($identifier) {
-                return new WC_Product($identifier);
+                try {
+                    $product = new WC_Product_Variation($identifier);
+                } catch (Exception $e) {
```

Comment:
brr...

Reply:
Helaas kan je aan de identifier niet zien of het een variatie is, en hoewel de variatie ook een WC_Product is, laat WooCommerce hem niet als zodanig instantiëren. Moet ik eerst dat uitvinden en dan de juiste class instantiëren? Dat leek me zoveel moeite en code tov de try catch...

Reply:
Opgelost met wc_get_product :-D